lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -1104,7 +1141,7 @@ sub process_and_run_upgrade_script {
         template => $template,
         no_auto_output => 1,
         format_options => {extension => 'sql'},
-        output_file => 'upgrade',
+        output_file => 'upgrade.sql',
```

Comment:
I thought that by setting the extension in the format options, the result would be a file *with* extension. Is this a bug elsewhere (which we may need to fix where it appears?)